packages/emptystate/src/react/index.tsx

```diff
@@ -0,0 +1,202 @@
+import { StyleAttribute, compose, css } from 'glamor'
```

Comment:
ugh....why are my git mv commands not working 🤮

Reply:
dunno.  

fwiw, I usually don't `git mv`.  I just `mv` and then `git add`, and then in the commit, it's seen as a move.